HW7_telbook/taks_1_collections.py

```diff
@@ -0,0 +1,91 @@
+
+print("Hi. This is a phone book. \nIf you have a new friend and you want to communicate with him, "
+      "you can save his contact. "
+      "\nFollow the menu instructions and make corrections to the book. GLHF!\n")
+
+print("1 - Add contact\n"
+      "2 - Number of records\n"
+      "3 - A list of all the names in the book\n"
+      "4 - Contact's detailed information\n"
+      "5 - Delete contact\n"
+      "0 - Exit the program")
+
+
+dict_contact = {}
+
+
+def add():
+    while True:
+        name = input("Enter the person's name: ")
+        number = input("Enter the person's number: ")
+        if len(name) > 0 and number.isdigit() and len(number) == 10:
+            if name in [i for i in dict_contact.keys()]:
```

Comment:
nit: there is no reason to create a list of keys
```suggestion
            if name in dict_contact:
```